apps/web/pages/auth/login.tsx

```diff
@@ -135,7 +148,7 @@ export default function Login({
                 <EmailField
                   id="email"
                   label={t("email_address")}
-                  defaultValue={router.query.email as string}
+                  defaultValue={totpEmail || (router.query.email as string)}
```

Comment:
When a totp code is needed in an external provider login use case, we still need the email to be filled to be sent in the form.